### DISCUSSION TITLE
Error when commenting lines via gcc and similar commands

### DISCUSSION THREAD

**Discussion by strbrgr** [LazyVim/LazyVim#777] [Q&A]
I am not able to type gcc to comment out a line anymore. Whenever I try to do this I will get the following error pop up. Is there something I am missing? 

`E5108: Error executing lua ....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:160: nvim_exec(): Vim(lua):E5108: Error executing lua ....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:392: attempt to call field 'get_filetypes' (a nil value)
stack traceback:
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:392: in function 'traverse'
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:403: in function 'get_commentstring'
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:345: in function 'make_comment_parts'
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:215: in function 'toggle_lines'
	[string ":lua"]:1: in main chunk
	[C]: in function 'cmd'
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:160: in function 'operator'
	[string ":lua"]:1: in main chunk
stack traceback:
	[C]: in function 'cmd'
	....local/share/nvim/lazy/mini.comment/lua/mini/comment.lua:160: in function 'operator'
	[string ":lua"]:1: in main chunk`

**Comment by folke** [comment]
Report in the mini.comment repo?

**Comment by strbrgr** [comment]
just a quick update if someone else runs into the same issue.

I added the latest stable branch of mini.nvim to my plugins in lazyvim:
`{ 'echasnovski/mini.nvim', version = '*' },`

using the main branch did not work and caused an error:
`{ 'echasnovski/mini.nvim', version = false },`

**Reply by redxtech** [reply]
that didn't fix it for me, i'll open an issue at mini.nvim
